vue-starter/README.md

```diff
@@ -16,10 +15,17 @@ To open the app in XCode, run: `npx cap open ios`.
 
 To open the app in Android Studio, run: `npx cap open android`.
 
+## Local Installation
+
+1. Clone the [Capacitor plugin project](https://github.com/pace/cloud-sdk-capacitor-plugin) into the **same** directory as the `vue-starter` repository.
```

Comment:
I just tried to follow these steps, but they are a bit too ambiguous for me.

When cloning this example app, the structure is: `cloud-sdk-ionic-example-app/vue-starter`

So which directory do you mean, when you say "the same directory as the vue-starter"? I would think it's within `cloud-sdk-ionic-example-app`, but looking at the `setup_locally.sh` script you probably mean the parent directory of the `cloud-sdk-ionic-example-app`?

Maybe we should add a tree representation here to make it perfectly clear.

Reply:
Oh right, `vue-starter` is wrong